app/Http/Livewire/SlackSettingsForm.php

```diff
@@ -45,17 +46,24 @@ public function mount() {
             "general"=> array(
                 "name" => trans('admin/settings/general.general_webhook'),
                 "icon" => "fab fa-hashtag",
-                "placeholder" => "",
+                "placeholder" => "Insert URL",
```

Comment:
I think we'd want that to be a translation string. <del>We might have one available already.</del> I [just added](#14158) one to `general.php`, so you should be able to use `trans('general.url')`.